src/core/api/repository_label.go

```diff
@@ -112,13 +87,62 @@ func (r *RepositoryLabelAPI) Prepare() {
 	}
 }
 
+func (r *RepositoryLabelAPI) hasPermission(action rbac.Action) bool {
```

Comment:
Merge the func `hasPermission` and `hasImageLabelPermission` into one?

Reply:
renamed `requireAccess`